SERVER_SIDE.md

```diff
@@ -71,6 +71,15 @@ var TomcatCookies = Cookies.withConverter({
 
 Alternatively, you can check the [Java Cookie](https://github.com/js-cookie/java-cookie) project, which integrates nicely with JavaScript Cookie.
 
+## Tomcat 8
```

Comment:
I wonder if we should use a new topic for a new Tomcat version. What if we change everything to a single "Tomcat" topic and create subtopics for 7.x and 8.x so we don't start cluttering the "root level" of the servers file. What do you think, does that makes sense?